app/components/trainees/confirmation/contact_details/view.rb

```diff
@@ -0,0 +1,54 @@
+module Trainees
+  module Confirmation
+    module ContactDetails
+      class View < GovukComponent::Base
+        include SanitizeHelper
+
+        attr_accessor :trainee
+
+        def initialize(trainee:)
+          @trainee = trainee
+          @not_provided_copy = I18n.t("components.confirmation.not_provided")
+        end
+
+        def address
+          return @not_provided_copy if trainee.locale_code.nil?
+
+          sanitize(uk_locale? ? uk_address : international_address)
+        end
+
+        def email
+          return @not_provided_copy unless trainee.email
+
+          trainee.email
+        end
+
+        def phone_number
+          return @not_provided_copy unless trainee.phone_number
+
+          trainee.phone_number
+        end
```

Comment:
https://dfe-twd-rttd-pr-85.herokuapp.com/trainees/2/contact-details/confirm

![image](https://user-images.githubusercontent.com/470137/95961854-8be72380-0dfd-11eb-9f03-a89a1e62a4c2.png)


1. Go to https://dfe-twd-rttd-pr-85.herokuapp.com/trainees/2/contact-details/edit 
2. Click continue with out any changes

Reply:
Good spot and should be fixed now